src/selectors/grid-cell-selectors.js

```diff
@@ -7,7 +7,7 @@ export const getTerrestrialCellData = createSelector(
   [selectCellData],
   cellData => {
     if (!cellData) return null;
-    return cellData.filter(c => c.ISMARINE === 0);
+    return cellData;
```

Comment:
Right now we are using two grids to create the aggregation of grid cells. The new 55km grid only stores terrestrial data. We are using the old grid to get only marine grid cells IDs (this situation will persists until MoL provides a dedicated marine grid). We do not have marine protection information right now (please @gretacv correct me if I'm wrong), so right now you need to filter terrestrial data before consuming it on the widget.

```js 
return cellData.filter(c => c.ID);
```
This snippet works because the old grid features do not have an `ID` attribute but a `CELL_ID` so getting only the ones with `ID` creates a terrestrial only subset.

Reply:
Thanks! :+1:  I wasn't fully aware of that